lib/ansible/cli/galaxy.py

```diff
@@ -569,7 +572,8 @@ def execute_info(self):
 
     def execute_install(self):
         """
-        uses the args list of roles to be installed, unless -f was specified. The list of roles
+        Install one or more roles, or one or more collections. You can pass in a list of roles or collections, or use the file
```

Comment:
This is a bit complicated but `ansible-galaxy role install` only installs roles while `ansible-galaxy collection install` only installs roles. The same method is called due to the weirdness of the arg spec setup so I'm not sure if you wanted to be more explicit with this info.